doc/api/test.md

```diff
@@ -514,7 +514,110 @@ test('spies on an object method', (t) => {
   assert.strictEqual(call.this, number);
 });
 ```
+## Mocking Timers
```

Comment:
Its fake timers under the mock module, the timers aren't really mocking anything afaiu?